client/dashboard/domains/domain-connection-setup/types.ts

```diff
@@ -0,0 +1,88 @@
+import {
+	DomainConnectionSetupMode,
+	DomainMappingSetupInfo,
+	DomainMappingStatus,
+} from '@automattic/api-core';
+
+export enum StepType {
```

Comment:
Let's avoid using enum.  Instead, we just need to use an object to map the constants. See p1754323932129889-slack-C0982082MSR